{% extends 'blog/base.html' %}

{% block content %}
<h1>META-Insight</h1>
<p>
    Metagenomic analysis is an innovative approach to interpreting complex biological information to understand the ecology, function, diversity, and interactions of microbial communities.
     Our goal is to introduce state-of-the-art analytical methods that enable accurate meta-omics analysis and provide meta-omics data derived from the accurate gut microbiome of healthy individuals.
     The key feature of our data analysis method is that it organizes microbial community information at the level of microbial genomes, mainly focusing on metagenome-assembled genomes (MAGs).
     Additionally, while metagenomes can offer potential functional information, the presence of specific genes does not necessarily imply their expression.
     By combining metatranscriptome and metaproteome data, we can confirm gene expression or suppression under specific conditions, allowing us to distinguish active microbes from inactive ones from a metabolic perspective.
      In particular, at each step of NGS meta-omics analysis, we introduce an analysis based on shotgun sequencing for metagenome, RNA-seq for metatranscriptome, and Ribo-seq data for metaproteome. In addition to meta-omics data analysis protocols, Meta-Insight provides precise meta-omics data of the gut microbiome in healthy individuals.
     Such data can be helpful not only in understanding the functions of the gut microbiome but also in serving as a control group for research on gut microbiomes related to diseases.
</p> <br><br>
<h1>CONTACT US</h1>
<p>
    Laboratory of Systems Microbiology, Gyeongsang National University <br>
    443 Building #351, 501 Jinju-daero, Jinju, Gyeongnam 52828, Republic of Korea<br>
    <a href="mailto:metainsight@gnu.ac.kr">HELP</a><br>
    E-mail: metainsight@gnu.ac.kr<br>
</p>

{% endblock content %}